src/Sulu/Bundle/ContactBundle/Import/Import.php

```diff
@@ -1078,37 +1153,47 @@ protected function addBankAccounts($data, $entity)
             }
 
             // create comments for old bank addresses
-            if ($this->checkData('blz' . $i, $data)) {
-                $noteTxt = '';
-                // check if note already exists, or create a new one
-                if (sizeof($notes = $entity->getNotes()) > 0) {
-                    $note = $notes[0];
-                    $noteTxt = $note->getValue() . "\n";
-                } else {
-                    $note = new Note();
-                    $this->em->persist($note);
-                    $entity->addNote($note);
-                }
-
-                $noteTxt .= 'Old Bank Account: ';
+            if ($this->checkData($prefix . 'blz', $data)) {
+                $noteTxt = 'Old Bank Account: ';
                 $noteTxt .= 'BLZ: ';
-                $noteTxt .= $data['blz' . $i];
+                $noteTxt .= $data[$prefix . 'blz'];
 
-                if ($this->checkData('accountNumber' . $i, $data)) {
+                if ($this->checkData($prefix . 'number', $data)) {
                     $noteTxt .= '; Account-Number: ';
-                    $noteTxt .= $data['accountNumber' . $i];
+                    $noteTxt .= $data[$prefix . 'number'];
                 }
-                if ($this->checkData('bank' . $i, $data)) {
+                if ($this->checkData($bankIndex, $data)) {
                     $noteTxt .= '; Bank-Name: ';
-                    $noteTxt .= $data['bank' . $i];
+                    $noteTxt .= $data[$bankIndex];
                 }
 
-                $note->setValue($noteTxt);
+                $this->appendToNote($entity, $noteTxt);
             }
         }
     }
 
     /**
+     * function either appends a text to the existing note, or creates a new one
+     * @param $entity The entity containing the note
+     * @param $text Text to append
+     * @internal param $data
+     */
+    protected function appendToNote($entity, $text)
+    {
+        $noteTxt = '';
+        if (sizeof($notes = $entity->getNotes()) > 0) {
+            $note = $notes[0];
```

Comment:
btw generally a better way is to do `$note = reset($notes)`, which will always return the first element of an array which prevents errors when the array does not have index `0`.